storybook/src/docs/usability.docs.mdx

```diff
@@ -0,0 +1,23 @@
+{/* @license CC0-1.0 */}
+
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Design Guidelines/Usability" />
+
+# Usability
+
+## Focus management
+
+In traditional multi-page applications (MPAs) moving to another page means resetting focus to the `<body>` element,
+or the first element with an `autofocus` attribute if there is one.
+For single-page applications (SPAs) this isn’t always done consistently.
+For example, Next.js sets the focus on a new page at roughly the same position as it was on the old page,
+resulting in an unexpected focus position.
```

Comment:
```suggestion
For example, Next.js sets the focus on a new page at roughly the same position as it was on the old page.
This can result in an unexpected focus position, e.g. when navigating between pages of a form.
```

Reply:
```suggestion
For example, Next.js sets the focus on a new page at roughly the same position as it was on the old page.
This can cause the focus to change unexpectedly, like when you move between pages in a form.
```

Reply:
> This can cause the focus to change unexpectedly, like when you move between pages in a form.

This kind of changes the meaning of that sentence.